corehq/apps/app_manager/templates/app_manager/partials/case_list.html

```diff
@@ -30,15 +30,17 @@
         <option value="no">Don't Use Case Tiles</option>
         <option value="yes">Use Case Tiles</option>
     </select>
-    <label class="checkbox" data-bind="visible: useCaseTiles() == 'yes'">
-        <input type="checkbox" data-bind="checked: persistTileOnForms">
-        Use this case list tile persistently in forms
-    </label>
-    <label class="checkbox" data-bind="visible: useCaseTiles() == 'yes' && persistTileOnForms()">
-        <input type="checkbox" data-bind="checked: enableTilePullDown">
-        Embed case details in case tile pull-down
-    </label>
     {% endif %}
+    <div data-bind="visible: useCaseTiles() == 'yes' || window.toggles.CASE_LIST_CUSTOM_XML">
```

Comment:
I made a chart to make it easier to think about :)
("-" means imposible case)
```
TILE    XML     useCaseTiles    show_boxes
0       0       0               0
0       0       1               -
0       1       0               1
0       1       1               -
1       0       0               0
1       0       1               1
1       1       0               1
1       1       1               1
```
So it looks like show_boxes should be true whenever the `CASE_LIST_CUSTOM_XML` toggle is true, and it should also be true when `CASE_LIST_TILE` is true and `useCaseTiles` is true. Since `useCaseTiles` can only be true when `CASE_LIST_TILE` is true, I think it boils down to the condition I currently have (`useCaseTiles() == 'yes' || window.toggles.CASE_LIST_CUSTOM_XML`).